php/7.4/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM php:7.4-fpm-alpine
+
+RUN apk upgrade --update \
+    && apk --update add --no-cache --virtual .phpize-deps $PHPIZE_DEPS imagemagick-dev libtool \
```

Could you put a bash into the container in a separate apk add command while also merging imagemagick from line 12 into it?

---

Please take a look if this works.
```suggestion
    && apk --update add --no-cache --virtual .phpize-deps $PHPIZE_DEPS imagemagick-dev libtool \
    && apk --update add --no-cache .imagick-runtime-deps imagemagick bash \
```

---

Also I assume line 12 will change to
```
&& apk add --no-cache --virtual .imagick-runtime-deps \
```

Am I assumed it correctly?

---

It can be deleted

---

Alright, I'll check it

---

Great! It worked! I'll push my modifications for a re-check.